app/worker/index.js

```diff
@@ -51,6 +53,46 @@ const setupRNDebugger = async message => {
     checkAvailableDevMenuMethods(modules, message.networkInspect);
     reportDefaultReactDevToolsPort(modules);
   }
+
+  const interval = setInterval(() => {
```

Comment:
Apollo client doesn't have any event emitted when it is set up. To overcome that I just poll the client till it appears.
It is not an optimal approach, but simple polling shouldn't be too much overhead